File: backend/src/main/java/tko/model/dto/nutrition/ProductForPageDTO.java
package tko.model.dto.nutrition;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ProductForPageDTO {
    private Long id;
    private String name;
    private Integer calories;
    private Integer percentOfTarget;
    private String unit;
    private BigDecimal portion;
}

File: app/src/main/java/tk/ssau/fitnesstko/model/dto/nutrition/ProductForPageDTO.kt
package tk.ssau.fitnesstko.model.dto.nutrition

import java.math.BigDecimal

data class ProductForPageDTO(
    val id: Long?,
    val name: String,
    val calories: Int,
    val percentOfTarget: Int,
    val unit: String,
    val portions: BigDecimal
)